### PR TITLE
fixes field-filtering during get_subs and ipaddr/port setting during cre...

### PR DESCRIPTION
**Pull request by yosemitebandit**
...ate_sub

tests were failing on the latest client build without these changes..I'm guessing it's due to changes in NM and SR

### CODE REVIEW

**Comment by yosemitebandit** [comment]
ptal

File: openbts/components.py [hunk 1]
```diff
@@ -86,25 +86,33 @@ def get_subscribers(self, imsi=None):
...
     try:
       response = self._send_and_receive(message)
       subscribers = response.data
     except InvalidRequestError:
-      subscribers = []
-    # Now attach the associated numbers, account balance and caller_id info.
+      return []
+
+    # We get back every field in the SR, most of which are not useful.  We will
+    # simplify each subscriber dict to show just a few ports.  And we will
```
Comment (shaddi) [inline openbts/components.py]: s/ports/parts?

**Comment by shaddi** [comment]
lgtm, though seems like the fix for setting the `ipaddr` and `port` is masking an underlying `subscriberRegistry` issue? (fine for now, probably worth flagging)

**Comment by yosemitebandit** [comment]
filed #14 to track

**Pull request merged**